kubesae/ansible/deploy.py

```diff
@@ -2,21 +2,31 @@
 
 
 @invoke.task
-def install_requirements(c):
+def install_requirements(c, requirements_file=None):
     """Install ansible-galaxy requirements.yml"""
+    if requirements_file is None:
+        if "requirements_file" in c.config:
+            requirements_file = c.config.requirements_file
+        else:
+            requirements_file = "requirements.yaml"
     with c.cd("deploy/"):
-        c.run("ansible-galaxy install -f -r requirements.yml -p roles/")
+        c.run(f"ansible-galaxy install -f -r {requirements_file} -p roles/")
 
 
 @invoke.task(pre=[install_requirements], default=True)
-def ansible_deploy(c, env=None, tag=None):
+def ansible_deploy(c, env=None, tag=None, play=None):
     """Deploy K8s application."""
     if env is None:
         env = c.config.env
     if tag is None:
         tag = c.config.tag
+    if play is None:
+        if "play" in c.config:
```

Comment:
Maybe namespace the config values, something like `c.config.ansible.deploy.play`?  At least for config values like this that are pretty specific to a particular task?